### PR TITLE
dev-python/jupyterhub: initial import

### PR DESCRIPTION
**Pull request by xmhd**
Previous attempt: #261 

As requested by @lemanchester 

This will require some testing.

### CODE REVIEW

File: dev-python/certipy/certipy-0.1.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+HOMEPAGE="https://github.com/LLNL/certipy"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD-2"
+KEYWORDS="~amd64"
+
+SLOT="0"
+
+RDEPEND="\
+	dev-python/jupyterhub[${PYTHON_USEDEP}] \
```
Comment (alip) [inline dev-python/certipy/certipy-0.1.3.ebuild:20]: nitpick: backslashes are not necessary.

File: dev-python/jupyter-telemetry/jupyter-telemetry-0.1.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+DESCRIPTION="Telemetry for Jupyter Applications and extensions"
+HOMEPAGE="https://github.com/jupyter/telemetry"
+SRC_URI="mirror://pypi/${PN:0:1}/jupyter_telemetry/jupyter_telemetry-${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+KEYWORDS="~amd64"
+
+SLOT="0"
+
+RDEPEND="\
```
Comment (alip) [inline dev-python/jupyter-telemetry/jupyter-telemetry-0.1.0.ebuild:19]: ditto.

File: dev-python/jupyter-telemetry/jupyter-telemetry-0.1.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+	dev-python/ruamel-yaml[${PYTHON_USEDEP}] \
+	dev-python/jsonschema[${PYTHON_USEDEP}] \
+	dev-python/python-json-logger[${PYTHON_USEDEP}] \
+	dev-python/traitlets[${PYTHON_USEDEP}] \
+"
+
+
+src_unpack() {
+	default
+	mv * ${P}
```
Comment (alip) [inline dev-python/jupyter-telemetry/jupyter-telemetry-0.1.0.ebuild:29]: 1. Can we please make the wildcard less specific? This scares me.
2. Can we please die if `mv` fails?

**Review by alip — APPROVED** [review]
LGTM.
Note, I have not tried building it.

**Comment by xmhd** [comment]
@alip fix'd , I done some copy pasta from overlays if you couldn't tell :-)

**Pull request merged**